lib/pymedphys/_streamlit/apps/metersetmap/main.py

```diff
@@ -249,10 +249,16 @@ def plot_and_save_results(
 
     diff = evaluation_metersetmap - reference_metersetmap
 
-    imageio.imwrite(reference_filepath, reference_metersetmap)
-    imageio.imwrite(evaluation_filepath, evaluation_metersetmap)
-    imageio.imwrite(diff_filepath, diff)
-    imageio.imwrite(gamma_filepath, gamma)
+    imageio.imwrite(
+        reference_filepath,
+        st_misc.normalize_and_convert_to_uint8(reference_metersetmap),
```

Comment:
won't normalising the reference and evaluation maps against themselves instead of against the union of the two result in images that don't visually represent them accurately?  What would happen if the evaluation meterset map was actually exactly 0.5 of the reference meterset map, across the board?  If the images are generated directly from the data rather than files as written, then I guess the question should be with respect to "whatever the written files are used for", but thinking of those as images, after self-normalisation, I'd expect the two to look exactly the same, while normalisation against the union would have one being half as bright as the other.

Reply:
Good catch, Thanks. I've adjusted the function so that the produced images will more accurately represent the deliveries.